psi4/share/psi4/databases/O24by5.py

```diff
@@ -0,0 +1,2264 @@
+BIND["%s-%s" % (dbse, "12-1.5")] = -521.59
+BIND["%s-%s" % (dbse, "12-2.0")] = -137.13
+# 13
+BIND["%s-%s" % (dbse, "13-0.9")] = -1431.56
+BIND["%s-%s" % (dbse, "13-1.0")] = -1983.78
+BIND["%s-%s" % (dbse, "13-1.2")] = -1352.50
+BIND["%s-%s" % (dbse, "13-1.5")] = -598.20
+BIND["%s-%s" % (dbse, "13-2.0")] = -206.86
+# 14
+BIND["%s-%s" % (dbse, "14-0.9")] = -2408.67
+BIND["%s-%s" % (dbse, "14-1.0")] = -2647.19
+BIND["%s-%s" % (dbse, "14-1.2")] = -1597.02
+BIND["%s-%s" % (dbse, "14-1.5")] = -678.35
+BIND["%s-%s" % (dbse, "14-2.0")] = -234.81
+# 15
+BIND["%s-%s" % (dbse, "15-0.9")] = -3965.73
+BIND["%s-%s" % (dbse, "15-1.0")] = -5014.58
+BIND["%s-%s" % (dbse, "15-1.2")] = -3810.80
+BIND["%s-%s" % (dbse, "15-1.5")] = -1673.44
+BIND["%s-%s" % (dbse, "15-2.0")] = -422.38
+# 16 Li - O2
+BIND["%s-%s" % (dbse, "16-0.9")] = -14.70
+BIND["%s-%s" % (dbse, "16-1.0")] = -36.31
+BIND["%s-%s" % (dbse, "16-1.2")] = -24.41
+BIND["%s-%s" % (dbse, "16-1.5")] = -7.19
+BIND["%s-%s" % (dbse, "16-2.0")] = -1.13
+# 17
+BIND["%s-%s" % (dbse, "17-0.9")] = -16.05
+BIND["%s-%s" % (dbse, "17-1.0")] = -50.41
+BIND["%s-%s" % (dbse, "17-1.2")] = -28.51
+BIND["%s-%s" % (dbse, "17-1.5")] = -7.71
+BIND["%s-%s" % (dbse, "17-2.0")] = -1.26
+# 18
+BIND["%s-%s" % (dbse, "18-0.9")] = -87.23
+BIND["%s-%s" % (dbse, "18-1.0")] = -103.77
+BIND["%s-%s" % (dbse, "18-1.2")] = -73.10
+BIND["%s-%s" % (dbse, "18-1.5")] = -25.73
+BIND["%s-%s" % (dbse, "18-2.0")] = -4.32
+# 19 H2O - O2 (gm)
+BIND["%s-%s" % (dbse, "19-0.9")] = -73.37
+BIND["%s-%s" % (dbse, "19-1.0")] = -223.22
+BIND["%s-%s" % (dbse, "19-1.2")] = -126.50
+BIND["%s-%s" % (dbse, "19-1.5")] = -35.99
+BIND["%s-%s" % (dbse, "19-2.0")] = -6.77
+# 20 Na - Li
+BIND["%s-%s" % (dbse, "20-0.9")] = -210.40
+BIND["%s-%s" % (dbse, "20-1.0")] = -259.27
+BIND["%s-%s" % (dbse, "20-1.2")] = -188.18
+BIND["%s-%s" % (dbse, "20-1.5")] = -75.17
+BIND["%s-%s" % (dbse, "20-2.0")] = -14.25
+# 21
+BIND["%s-%s" % (dbse, "21-0.9")] = -187.20
+BIND["%s-%s" % (dbse, "21-1.0")] = -266.29
+BIND["%s-%s" % (dbse, "21-1.2")] = -132.52
+BIND["%s-%s" % (dbse, "21-1.5")] = -36.14
+BIND["%s-%s" % (dbse, "21-2.0")] = -6.37
+# 22
+BIND["%s-%s" % (dbse, "22-0.9")] = -366.42
+BIND["%s-%s" % (dbse, "22-1.0")] = -658.94
+BIND["%s-%s" % (dbse, "22-1.2")] = -415.05
+BIND["%s-%s" % (dbse, "22-1.5")] = -140.79
+BIND["%s-%s" % (dbse, "22-2.0")] = -31.54
+# 23
+BIND["%s-%s" % (dbse, "23-0.9")] = -826.91
+BIND["%s-%s" % (dbse, "23-1.0")] = -1018.90
+BIND["%s-%s" % (dbse, "23-1.2")] = -790.24
+BIND["%s-%s" % (dbse, "23-1.5")] = -341.29
+BIND["%s-%s" % (dbse, "23-2.0")] = -74.18
+# 24
+BIND["%s-%s" % (dbse, "24-0.9")] = -8922.76
+BIND["%s-%s" % (dbse, "24-1.0")] = -10324.99
+BIND["%s-%s" % (dbse, "24-1.2")] = -7656.09
+BIND["%s-%s" % (dbse, "24-1.5")] = -3703.01
+BIND["%s-%s" % (dbse, "24-2.0")] = -1641.58
+
+# scale back to kcal/mol
+for key, value in BIND.items():
+    BIND[key] = value / constants.kcalmol2wavenumbers
+
+# <<< Comment Lines >>>
+TAGL = {}
```

Comment:
These are very repetitive dictionary-filling statements. Can I convince you to rewrite this as a set of for loops? One over `DIST` and one over `enumerate(new_array_containing_dimer_names, start=1)`. The same goes in the other file.

Reply:
I'll try to set it up, shortly.